### DISCUSSION TITLE
Accidentally removed an email

### DISCUSSION THREAD

**Discussion by megamayoy** [community/community#21661] [New to GitHub]
<p>I changed my primary email to another and I accidentally deleted it so I tried to add it again. When I try to do so, it shows the error message “Email already in use”. I don’t know why this is showing. I don’t have any other Github account with this email. Any help?</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>In my experience, I have never seen that “Email already in use” be wrong so I’m going to ask you to humour me :slight_smile:</p>
<p>Please either log out of GitHub entirely or use a separate browser, and then enter your email address here:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/password_reset" target="_blank" rel="noopener">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/11161530/181066067-4cc13912-fad4-4f68-b155-18529997839c.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://github.com/password_reset" target="_blank" rel="noopener">Build software better, together</a></h3>

<p>GitHub is where people build software. More than 56 million people use GitHub to discover, fork, and contribute to over 100 million projects.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>If you are using a separate browser for this task, make sure you copy and paste the password reset link from your email into the correct browser window - not one where you are logged in already.</p>
<p>If this doesn’t log you into a different account, then please <a href="https://support.github.com/contact">open a ticket</a> so that support can discuss your account in more detail with you (in private). Make sure to tell them that you’ve tried this, and any error messages you may have seen - that will speed up the resolution!</p>
